packages/oruga-next/src/components/dropdown/Dropdown.vue

```diff
@@ -500,6 +512,19 @@ const rootClasses = defineClasses(
         null,
         computed(() => isMobileModal.value && !hoverable.value),
     ],
+    [
+        "rootPositionClass",
+        "o-drop--position-",
+        autoPosition,
+        computed(() => !!autoPosition.value),
+    ],
+    [
+        "rootActiveClass",
+        "o-drop--active",
+        null,
+        computed(() => isActive.value || props.inline),
+    ],
+    ["hoverableClass", "", null, computed(() => hoverable.value)],
```

Comment:
`hoverable` is already a computed prop so `["hoverableClass", "", null, hoverable]` might be better

Reply:
Good point, that's redundant. Fixed.